app/src/main/java/com/strv/movies/ui/moviedetail/MovieDetailScreen.kt

```diff
@@ -82,29 +84,49 @@ fun MovieTrailerPlayer(videoId: String, progressSeconds: MutableState<Float>) {
 
 @Composable
 fun MoviePoster(movie: MovieDetail) {
-    AsyncImage(
-        model = "https://image.tmdb.org/t/p/w500${movie.posterPath}",
-        contentDescription = stringResource(id = R.string.movie_image),
-        modifier = Modifier
-            .padding(top = 16.dp)
-            .size(120.dp)
-    )
+    Column {
+
+        AsyncImage(
+            model = "https://image.tmdb.org/t/p/w500${movie.posterPath}",
+            contentDescription = stringResource(id = R.string.movie_image),
+            modifier = Modifier
+                .padding(top = 16.dp)
+                .size(120.dp),
+        )
+        Row {
+            Text(
+                movie.title,
+                fontWeight = FontWeight.Bold,
+                textAlign = TextAlign.Center,
+                fontSize = 10.sp,
+                color = Color.Red
+            )
+        }
+    }
 }
 
 @Composable
 fun MovieInfo(movie: MovieDetail) {
     Column {
         Text(
             movie.title,
-            modifier = Modifier.padding(top = 16.dp, end = 16.dp),
+            modifier = Modifier
+                .padding(start = 10.dp, top = 16.dp, end = 10.dp),
             fontWeight = FontWeight.Bold,
```

Comment:
It is good to use Material Typography system. Instead of defining fontWeight and fontSize try to find suitable style here: https://material.io/design/typography/the-type-system.html#type-scale
Than you can use it like this:
`style = MaterialTheme.typography.subtitle2`